### DISCUSSION TITLE
Obsidian的移动端/手机端解决方案？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#37] [Q&A常见问题] [answered]
我目前主要是通过坚果云直接同步obsidian的文件夹，在坚果云手机端上查看。大家来分享一下你的做法吧

坚果云
* 优点: 有坚果云Markdown，可以查看、编辑markdown文件
* 缺点: 搜索功能需要付费使用....有很多文件的时候无法搜索等于查看不了...

**Comment by alanjustalan** [comment]
我主要用易码，markor也不错

**Reply by lefinite** [reply]
目前用着易码最佳

**Reply by zhixiangCai** [reply]
易码是什么软件呀？可以给个链接吗，我在Appstore 里没有搜到

**Reply by zraistlin61** [reply]
> 易码是什么软件呀？可以给个链接吗，我在Appstore 里没有搜到

安卓软件。可以通过酷市场下载。IOS不清楚。

**Reply by lnuiv** [reply]
> > 易码是什么软件呀？可以给个链接吗，我在Appstore 里没有搜到
> 
> 安卓软件。可以通过酷市场下载。IOS不清楚。

同用着易码，但同步到obsidian好像有点问题……在易码写的文件内容最前面都会有个红点。网上好像都没看到说这个问题，是个例还是可以解决的通病……

**Reply by lefinite** [reply]
红点问题我也遇到了，不知道是什么原因

**Comment by cyddgh** [comment]
听说ios上可以使用1Writer
![image](https://user-images.githubusercontent.com/10444611/103150320-e7df0880-47ad-11eb-8b8e-aec65a0da51b.png)

**Reply by cyddgh** [reply]
听说drafts也不错。
但我考虑是不是还能尝试一下logseq
logseq也能读md，本身也是web端

**Comment by Quorafind** [comment]
开发者已经在着手进行手机版 Obsidian 的开发了

**Reply by makerjackie** [reply]
great，希望不翻墙能下载/同步

**Reply by Quorafind** [reply]
应该是会支持本地文件加载，也就是可以用坚果云同步到手机本地

**Reply by onedayluckys** [reply]
Obsidian安卓不支持手机SD存储卡，目前这版只能存储到手机内存中。

**Comment by Jerry3062** [comment]
我也没想到好的办法，平时直接拷贝到手机目录，这样可以用Obsidian打开，完了备份一个到github。

**Comment by fred1357944** [comment]
建議重度使用者能夠購買sync服務，個人使用經驗不錯👍
已經同步將Macbook多個Vault同步到ipad&iphone。
可參考如何設置https://youtu.be/1doCZsDK2ag

**Comment by ViZAlice** [comment]
用icloud呀，obsidian可以直接用icloud上的库，然后pc上也直接打开云盘上的库就行了。

**Comment by tidalwright** [comment]
安卓与Windows可以使用onedrive作为云存储，不过安卓端需要借助foldersync绑定onedrive来进行同步

**Comment by imshenwei** [comment, marked as answer]
[Obsidian同步方案(win+android)](https://blog.csdn.net/Brain8Storm/article/details/122518821)
